baseline/ica/ica.py

```diff
@@ -15,9 +15,9 @@
 from ica.jade import JADE
 from lib.norms import Norm
 
-TEST = True
+TEST = False
```

Comment:
Any reason why this is set to false now?

Reply:
He wanted to run it in train. Probably fine